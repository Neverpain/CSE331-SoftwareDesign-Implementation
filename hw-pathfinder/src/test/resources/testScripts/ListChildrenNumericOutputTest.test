# Tests that the ListChildren method in the driver compares edges numerically
# , outputs them out

CreateGraph g

AddNode g A
AddNode g B

AddEdge g A B 5.0
AddEdge g A B 1.0

ListChildren g A