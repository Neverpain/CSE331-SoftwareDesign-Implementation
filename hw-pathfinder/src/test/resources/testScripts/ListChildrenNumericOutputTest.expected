# Tests that the ListChildren method in the driver compares edges numerically
# , outputs them out

created graph g

added node A to g
added node B to g

added edge 5.000 from A to B in g
added edge 1.000 from A to B in g

the children of A in g are: B(1.000) B(5.000)